src/main/java/org/openstreetmap/atlas/geography/sharding/DynamicTileSharding.java

```diff
@@ -389,13 +389,16 @@ protected int onRun(final CommandMap command)
         {
             final StringList split = StringList.split(line, ",");
             final SlippyTile tile = SlippyTile.forName(split.get(0));
-            counts.put(tile, Integer.valueOf(split.get(1)));
+            counts.put(tile, Long.valueOf(split.get(1)));
             zoom = tile.getZoom();
             if (++counter % READER_REPORT_FREQUENCY == 0)
             {
                 logger.info("Read counts for {} zoom level {} tiles.", counter, zoom);
             }
         }
+        final HashMap<SlippyTile, Long> allCounts = calculateTileCountsForAllZoom(maximumZoom - 2,
```

Comment:
Can you add a code comment here to explain what the `2` means?

Reply:
Added!